notebooks/enterprise-search/app-search-engine-exporter.ipynb

```diff
@@ -0,0 +1,894 @@
+        "One thing to notice is how App Search uses [multi-fields](https://www.elastic.co/guide/en/elasticsearch/reference/current/multi-fields.html) in Elasticsearch that allow for quickly changing the field type in App Search without requiring reindexing by creating subfields for each type of supported field:\n",
+        "\n",
+        "<details>\n",
+        "  <summary>An example schema can be found by clicking here</summary>\n",
+        "\n",
+        "```json\n",
+        "\"[APP_SEARCH_ENGINE_FIELD_NAME]\": {\n",
+        "  \"type\": \"text\",\n",
+        "  \"fields\": {\n",
+        "    \"date\": {\n",
+        "      \"type\": \"date\",\n",
+        "      \"format\": \"strict_date_time||strict_date\",\n",
+        "      \"ignore_malformed\": true\n",
+        "    },\n",
+        "    \"delimiter\": {\n",
+        "      \"type\": \"text\",\n",
+        "      \"index_options\": \"freqs\",\n",
+        "      \"analyzer\": \"iq_text_delimiter\"\n",
+        "    },\n",
+        "    \"enum\": {\n",
+        "      \"type\": \"keyword\",\n",
+        "      \"ignore_above\": 2048\n",
+        "    },\n",
+        "    \"float\": {\n",
+        "      \"type\": \"double\",\n",
+        "      \"ignore_malformed\": true\n",
+        "    },\n",
+        "    \"joined\": {\n",
+        "      \"type\": \"text\",\n",
+        "      \"index_options\": \"freqs\",\n",
+        "      \"analyzer\": \"i_text_bigram\",\n",
+        "      \"search_analyzer\": \"q_text_bigram\"\n",
+        "    },\n",
+        "    \"location\": {\n",
+        "      \"type\": \"geo_point\",\n",
+        "      \"ignore_malformed\": true,\n",
+        "      \"ignore_z_value\": false\n",
+        "    },\n",
+        "    \"prefix\": {\n",
+        "      \"type\": \"text\",\n",
+        "      \"index_options\": \"docs\",\n",
+        "      \"analyzer\": \"i_prefix\",\n",
+        "      \"search_analyzer\": \"q_prefix\"\n",
+        "    },\n",
+        "    \"stem\": {\n",
+        "      \"type\": \"text\",\n",
+        "      \"analyzer\": \"iq_text_stem\"\n",
+        "    }\n",
+        "  },\n",
+        "  \"index_options\": \"freqs\",\n",
+        "  \"analyzer\": \"iq_text_base\"\n",
+        "}\n",
+        "```\n",
+        "</details>\n",
+        "\n",
+        "In our case we can assume that we have a well established schema and we do not need to use all multi-fields.\n",
+        "\n",
+        "We can retrieve the field types of an App Search engine using the [Schema API](https://www.elastic.co/guide/en/app-search/current/schema.html) and then construct our mapping."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "# define SOURCE_INDEX and DEST_INDEX which we will continue to reuse; feel free to adjust DEST_INDEX\n",
```

Comment:
Would it make sense to get the value of `DEST_INDEX` as an input parameter?

Reply:
Meh - I'm on the fence about that... I think it's enough to have it as a variable, but, in the markdown above that definition, I'll call it out a bit better.